### PR TITLE
Feat: P0 authentication (login & signup)

### PR DESCRIPTION
**Pull request by Naveenchandar**
LOGIN
Screenshot 2022-02-12 at 10 01 01 AM

SIGN UP
Screenshot 2022-02-12 at 10 02 28 AM

### CODE REVIEW

**Review by anupkgurung — APPROVED** [review]
LGTM.

File: navbar.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.nav_group {
+    width: 100%;
+    margin: 0;
+    border-radius: unset;
+}
+
+.nav_actions>span {
+    padding: 0;
+    color: #fff;
```
Comment (rohanmathur91) [inline navbar.css]: have a variable for color

File: style.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+:root {
+    font-size: 1rem;
+}
+
+body {
+    background-color: #F7F7F7;
+}
```
Comment (rohanmathur91) [inline style.css]: same as above

File: style.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+#login span, #login label {
+    font-size: 1rem;
+}
+
+.login_body {
+    width: 30%;
+    background-color: #ffffff;
+}
```
Comment (rohanmathur91) [inline style.css]: same as above

**Review by rohanmathur91 — APPROVED** [review]
Suggested few changes, rest looks good

**Pull request merged**